src/styles/style.css

```diff
@@ -0,0 +1,112 @@
+@import "./stars.css";
+
+.page {
+  font-family: "Roboto", sans-serif;
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+}
+
+.page-content {
+  margin: 0;
+  padding: 0;
+
+  color: var(--main-accent);
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  padding: 0 3px;
+  vertical-align: bottom;
+}
+
+.card__title {
+  margin: 40px 0 0;
+
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card__code {
+  margin: 4px 0 0;
+
+  line-height: 14px;
+  color: var(--secondary);
+}
+
+.card__rewiews {
+  display: flex;
+  margin-top: 16px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__rewiews-count {
+  margin: 0;
+}
+
+.card__rewiews-count,
+.card__code {
+  font-size: 10px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-title {
+  margin: 24px 0 0;
+
+  font-size: 12px;
+  color: var(--secondary);
+}
+
+.card__price-value {
+  margin: 24px 0 0;
```

Comment:
will beter to use margin-bottom, add margin-bottom to prev element
add reset.css and reset margin there